apps/nextjs-website/src/components/molecules/ApiSection/ApiSection.tsx

```diff
@@ -108,7 +109,7 @@ const ApiSection = ({
   const styles = getStyles(palette);
 
   return (
-    <>
+    <Box sx={{ marginTop: { xs: '62px', sm: '78px' } }}>
```

Comment:
Just a curiosity here: wouldn't be better to add some margin to the header? in this case every other component, below the header, have the same distance.

am i right? 🤔